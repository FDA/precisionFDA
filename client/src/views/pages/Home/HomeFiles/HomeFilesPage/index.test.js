import React from 'react'
import { shallow } from 'enzyme'

import { HomeFilesPage } from './index'


describe('HomeFilesPage test', () => {
  it('should render', () => {
    const component = shallow(<HomeFilesPage />)

    expect(component).toMatchSnapshot()
  })
})
