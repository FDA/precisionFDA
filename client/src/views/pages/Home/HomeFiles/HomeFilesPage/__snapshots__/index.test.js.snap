// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeFilesPage test should render 1`] = `
<HomeLayout>
  <div
    className="home-page-layout__header-row"
  >
    <div
      className="home-page-layout__actions"
    >
      <Button
        onClick={[Function]}
        type="primary"
      >
        <span>
          <Icon
            icon="fa-plus"
          />
            Add Folder
        </span>
      </Button>
      <Button
        type="primary"
      >
        <span>
          <Icon
            icon="fa-plus"
          />
            Add Files
        </span>
      </Button>
    </div>
    <div
      className="home-page-layout__actions--right"
    >
      <Connect(ActionsDropdown)
        deleteFiles={[Function]}
        files={Array []}
      />
    </div>
  </div>
  <div
    className="pfda-padded-t20"
  >
    <Connect(HomeFilesTable)
      files={Array []}
      handleFilterValue={[Function]}
    />
  </div>
  <div
    className="home-page-layout__upload-files-modal"
  >
    <UploadModal
      onClose={[Function]}
      scope="private"
      title="Upload files to Private Area"
    />
  </div>
</HomeLayout>
`;
